ocaml-lsp-server/test/e2e-new/action_inline.ml

```diff
@@ -270,18 +272,15 @@ let _ =
     {|
     let _ =
       let f (type a) (x : a) = x in
-      ((fun (type a) -> fun (x : a) -> x) 0) |}]
+      ((fun (type a) (x : a) -> x) 0) |}]
 
 let%expect_test "" =
   inline_test {|
 let _ =
   let $f : int -> int = fun x -> x in
   f 0
 |};
-  [%expect {|
-    let _ =
-      let f : int -> int = fun x -> x in
-      (0) |}]
+  [%expect {| |}]
```

Comment:
What happened in this test?

Reply:
cc @jfeser 

It looks like this does not fit the description you made of the "less agressive" beta reduction:
> The new version of beta reduction is more correct but less aggressive. It only reduces fully applied functions and does not attempt to reduce functions with labeled parameters.

Reply:
Give me a chance to fix the nix compatibility before you merge though.